sass/components/_modals.scss

```diff
@@ -0,0 +1,209 @@
+@import "../abstracts/_variables.scss";
+@import "../base/_animations.scss";
+
+.modals {
+  margin-top: 10rem;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+// BUTTON STYLES
+.button {
```

Comment:
This is a pretty generic class, consider scoping it somehow if you want specific styling only in modals. 😉